examples/web-cli/src/App.tsx

```diff
@@ -232,6 +231,7 @@ function App() {
         websocketUrl={currentWebsocketUrl}
         resumeOnReload={true}
         enableSplitScreen={true}
+        showSplitControl={true}
```

Comment:
This is true by default, but is included deliberately here to show the presence of the prop since this is an example.